node/src/cli/commands/login.command.ts

```diff
@@ -32,13 +33,15 @@ export class LoginCommand {
     protected clientId: string;
 
     private ssoRedirectUri: string = null;
+    private logService: ConsoleLogService;
 
     constructor(protected authService: AuthService, protected apiService: ApiService,
         protected i18nService: I18nService, protected environmentService: EnvironmentService,
         protected passwordGenerationService: PasswordGenerationService,
         protected cryptoFunctionService: CryptoFunctionService, protected platformUtilsService: PlatformUtilsService,
         clientId: string) {
         this.clientId = clientId;
+        this.logService = new ConsoleLogService(false);
```

Comment:
This should be passed in as a parameter so the application can determine if it's a dev instance and alter log filtering as appropriate.

I missed this in your work on the `unlock` command, sorry 😞

Reply:
Will make the appropriate changes and then create a new PR for the bitwarden cli as well